gettsim/data/wohngeld.yaml

```diff
@@ -24,6 +24,7 @@ koeffizienten_berechnungsformel:
     note: The keys from 1 to 12 below refer to the household size.
     1984-01-01:
       note: Parameter aus Regressionsanalyse der Wohngeldtabellen
+      reference:
```

Comment:
You don't have to delete, but we don't need to insert empty keys!

Reply:
I'm working on `paramviz` (#122) in parallel and I'd like to have references everywhere :)

Reply:
Please don't. It is very easy to work around on the code side. That is where it should go. `dict.get("reference", None)` out of the top of my head.

Reply:
fair enough